client/src/profile/ProfileContainer.js

```diff
@@ -95,16 +95,18 @@ class ProfileContainer extends React.Component {
 }
 
 ProfileContainer.propTypes = {
-  profile: PropTypes.object,
-  user: PropTypes.object.isRequired,
+  accessToken: PropTypes.string,
+  email: PropTypes.string,
```

Comment:
This is the issue with our current requireAuth solution. Email and AccessToken should be required for this component but I think you are right they can't be because otherwise we get errors when someone unauthenticated goes to /profile. You should make email not required in UserForm as well.